benchmarks/vision_bench.py

```diff
@@ -44,10 +44,10 @@ def run(self, cache, dataset_name):
         check = check_class()
         try:
             if isinstance(check, SingleDatasetCheck):
-                check.run(train_ds, train_predictions=train_pred, device=device)
+                check.run(train_ds, train_predictions=train_pred, device=device, n_samples=1_000)
             elif isinstance(check, TrainTestCheck):
                 check.run(train_ds, test_ds, train_predictions=train_pred,
-                          test_predictions=test_pred, device=device)
+                          test_predictions=test_pred, device=device, n_samples=1_000)
```

Comment:
these two n_samples lines override check parameters and can confuse in benchmark results, remove them